eloquentjs/11_1_quiet_times/quiet_times.js

```diff
@@ -0,0 +1,41 @@
+const { time } = require('console');
+const { generateLogs } = require('./src/generateLogs');
+const { textFile } = require('./src/textFile');
+const { fileToArray } = require('./src/fileToArray');
+
+/**
+ * Returns promise of hourly security camera activity recorded for given day of week (0-6)
+ * @param {*} day 
+ * @returns 
+ */
+async function activityTable(day) {
+    const logs = await Promise.all(
+        fileToArray(await textFile('camera_logs.txt'))
```

Comment:
I'd split this inner expression out into it's own const called something like `logPromises`. Nesting it here makes things a little hard to read.